Update plan 4 index.md (#79)
steno-aarhus/research: onboarding/getting-started-plan-4/index.md

---

::: callout-important

> This document is specific to SDCA employees working on Level 4 in Forum. For more onboarding information please read through the [SDCA Onboarding documents for all employees](https://steno-aarhus.github.io/research/onboarding.html).

Reviewer: ```suggestion
This document is specific to SDCA employees working on Level 4 in Forum. For more onboarding information please read through the [SDCA Onboarding documents for all employees](/onboarding.md).
```

are logged in via Citrix and have opened Microsoft Edge.
:::

> ## AU Printer
> Read about the AU printer under [General practicalities](https://steno-aarhus.github.io/research/onboarding/practicalities/)

Reviewer: ```suggestion
## AU Printer

Read about the AU printer under [General practicalities](/onboarding/practicalities/index.md)
```

> ## Office supplies/materials
> Read about Office supplies and more under [General practicalities](https://steno-aarhus.github.io/research/onboarding/practicalities/)

Reviewer: ```suggestion
## Office supplies/materials

Read about Office supplies and more under [General practicalities](/onboarding/practicalities/index.md)
```

| 8200 Aarhus N
| Att: YOUR NAME

Remember that your name must be stated when ordering.

### Address for Visitors:

| SDCA Forskning, plan 4
| Palle Juul Jensens Boulevard 11
| Indgang A
| 8200 Aarhus N

> ## Parking
> Read about Parking at AUH [here](https://steno-aarhus.github.io/research/onboarding/parking-at-auh/).

Reviewer: ```suggestion
## Parking

Read about Parking at AUH [here](/onboarding/parking-at-auh/index.md).
```

-   Plan 2 A201-028 (hallway)
-   Plan 3 A301-040 (niche)

> ## Canteen
> Read about the canteens at AUH under [General practicalities](https://steno-aarhus.github.io/research/onboarding/practicalities/)

Reviewer: ```suggestion
## Canteen

Read about the canteens at AUH under [General practicalities](/onboarding/practicalities/index.md)
```
